mindsdb/mindsdb
# [1/5] Docs: Add initial configuration for docs-v2 and first three pages

# Summary

This PR is the first out of the five that setups up V2 of the MindsDB documentation! The intended end result can be viewed at [mindsdb.mintlify.com](https://mindsdb.mintlify.com), but the endeavour is broken into smaller PRs for easier reviewing.

## Expectations 🎯
- All pages shall have the same content as the current docs (more features, now enabled by V2, will be introduced in later PRs)
- All pages must be available at the identical URLs and paths as the current docs
- All assets (images, and other files), are the same as the current docs

## What is added [1/5]
- Folder for `docs-v2`
- Documentation configuration at `mint.json`
- First three pages `Quickstart`, `What is MindsDB`, and `Getting Started`
- All assets to be used for the entire docs as it's difficult to separate them in bite-sized PRs due to the sheer number of them (currently duplicated from `/docs/mindsdb-docs/docs/assets`)

<img width="1156" alt="mindsdb-1" src="https://user-images.githubusercontent.com/44352119/201017170-4e0a4734-18cb-4f12-9058-c06e713fdf22.png">

## Testing Instructions

Download the Mintlify CLI using the following command

```bash
npm i mintlify -g
```

Run the following command inside `/docs-v2` to preview changes locally.

```bash
mintlify dev
```

Check to see that everything runs properly and all content matches the expectations mentioned above and all other specifications.

mindsdb
